EIPS/eip-3382.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 3382
+title: Hardcoded Block Gas Limit
+author: Philippe Castonguay (@PhABC)
+discussions-to: https://ethereum-magicians.org/t/eip-3382-hardcoded-gas-limit
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-13
+---
+
+## Simple Summary
+
+Hardcode the block gas limit (known as *block gas target* under [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)) to `12,500,000` gas per block.
+
+## Abstract
+
+Ethereum's block gas limit is currently dictated by block producers and is not enforced when validating blocks in clients. This EIP proposes to hardcode the block gas limit to `12,500,000`.
+
+## Motivation
+
+Both Ethereum's Proof of Work and Proof of Stake designs assume that block producers are financially rational, but does not assume block producers to be benevolent. There is one exception however, and it is when block producers choose the gas limit of a block where it is assumed that block producers care about the long term health and decentralisation of the chain. Indeed, the block gas limit is one of the only parameters in Ethereum that is not dictated by node consensus, but instead is chosen by block producers. This decision was initially made to allow urgent changes in the block gas limit if necessary. Both drastically increasing or decreasing this parameter could have serious consequences that may not be desired. It is therefore a critical parameter that should require node consensus to avoid any sudden harmful change imposed by a small number of actors on the rest of the network.
+
+## Specification
+Refer to `gasLimit` as `gasTarget` post EIP-1559.
+
```

Comment:
Write this either under the assumption of EIP-1559 or not.  We can always change the wording one way or the other at the last minute should plans change since that would be a non-normative change.  I recommend writing this under the assumption of EIP-1559, since that seems very likely to land in London.
```suggestion
```

Reply:
I initially used gasTarget across the entire EIP, however as @lightclient pointed out, these two EIPs are independant so it might be clearer if we use the current term.

Reply:
I agree with @lightclient that this EIP is independent of 1559 and should be written as such.  However, you should write a single EIP as an assertion of a change, without conditionals, which means you have to choose one wording or the other.  In this case, I think EIP-1559 is more likely to be included in London than not, so the "safer" bet is to use "gas target" wording as that minimizes the chances that you will need to update the EIP later should the environment change.